### PR TITLE
ci: prevent `benchmarks` job for running on an action for fork

### PR DESCRIPTION
**Pull request by derevnjuk**
## Information

Type | Breaking change
---|---
Chore | No

****

See details in actions/github-script#42

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -104,6 +104,7 @@ jobs:
         run: yarn build
 
   benchmarks:
+    if: github.repository_owner == 'tsedio'
```
Comment (Romakita) [inline .github/workflows/build.yml]: haaaaa perfect ^^

**Review by Romakita — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (derevnjuk) [reply .github/workflows/build.yml]: Unfortunately, this doesn't work as expected) I will try to find out another solution. Github allows managing [permissions](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions) on job and workflow level, but I'm not sure there is a way to use them as an expression

**Pull request merged**